elasticdl/python/allreduce/pytorch_controller.py

```diff
@@ -75,4 +81,26 @@ def restore(self):
         time.sleep(3)
         # Call `load_state_dict` to reset the state of Horovod optimizer
         self._optimizer.load_state_dict(self._optimizer.state_dict())
+        self._optimizer.zero_grad()
         self._rendezvous_manager.init_horovod_if_needed()
+
+    def reset_backward_passes_per_step(self):
```

Comment:
So this method plays the same role of AdjustBackwardPassesPerStepHook in TF1.x?

Reply:
Yes.